test/controllers/DuplicateClaimYearSelectionControllerSpec.scala

```diff
@@ -282,7 +235,7 @@ class DuplicateClaimYearSelectionControllerSpec extends SpecBase with MockitoSug
 
     "redirect to Technical Difficulties for a POST when ProfessionalBodiesService fails" in {
 
-      when(mockProfessionalBodiesService.professionalBodies(any())).thenReturn(Future.failed(new Exception))
+      when(mockProfessionalBodiesService.professionalBodies).thenReturn(List.empty)
```

Comment:
I don't think we need this test, it's saying that the service successfully reads the file and but for some reason it's empty. We already have a test that checks that the list is non-empty so this scenario is invalid for the system